client/src/components/editor/LinkExternalHref.js

```diff
@@ -3,23 +3,24 @@ import { IconNames } from "@blueprintjs/icons"
 import PropTypes from "prop-types"
 import React from "react"
 
-const LinkExternalHref = ({ url, children }) => {
+const LinkExternalHref = ({ url, children, attributes }) => {
   return (
-    <a href={url} target="_blank" rel="noreferrer">
-      {children}{" "}
+    <a href={url} target="_blank" {...attributes} rel="noreferrer">
+      {children}
       <Icon
         icon={IconNames.SHARE}
         intent={Intent.PRIMARY}
         size={IconSize.STANDARD * 0.75}
-        style={{ paddingBottom: "5px" }}
+        style={{ marginLeft: 2, paddingBottom: "5px" }}
```

Comment:
Should probably be `marginLeft: "2px", …` (anything other than 0 needs a unit).

Reply:
OK